src/lyra/tests/usage/example_numpy.py

```diff
@@ -0,0 +1,24 @@
+from typing import *
+from numpy import ndarray, arange
+
+alist: List[str] = ['a']
+#alist: List[str] = alist.append('append_test')
+array: ndarray = [1]
```

Comment:
This is not how numpy arrays are created! It should look like `array: ndarray = numpy.array([1])`. We might also want to support other common array creation routines.